packages/components/src/theme/ComposerTheme/types.ts

```diff
@@ -1,5 +1,13 @@
+import { SystemStyleObject } from '@chakra-ui/react';
+import { ComponentStyleConfig } from '@chakra-ui/theme';
 import { Colors } from './Colors';
 
 export interface Theme {
   colors: Colors;
+  components: {
+    [key: string]: ComponentStyleConfig;
```

Comment:
Is this going to work for custom components like TypePairing?

Reply:
yes, it should work for all components